src/sentry/runner/initializer.py

```diff
@@ -150,11 +150,20 @@ def bootstrap_options(settings, config=None):
                 setattr(settings, options_mapper[k], v)
 
 
+def bootstrap_logging():
+    from sentry import options
```

Comment:
We should document why we have to do it this way and what is going on here since next month, we'll have no idea what's going on.